README.md

```diff
@@ -7,9 +7,85 @@ Table of Contents
 =================
 
 * [Name](#name)
+* [Synopsis](#synopsis)
 * [Description](#description)
 * [Copyright & License](#copyright--license)
 
+Synopsis
+========
+
+Basic usage
+
+```
+
+# set variable in ob file scope.
+foo = bar;  we create a variable foo which value is bar. variable in ob is string 
+replace , op will render variable like some template engine.
```

Comment:
Well, it's not really like template engine since template engines never care about the background but opsboy understands everything, both foreground and background. This is *variable interpolation*.